modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,69 +21,103 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
 endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
-= Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
 
-You can customize the install-config.yaml file to specify more details about
-your {product-title} cluster's platform or modify the values of the required
-parameters.
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[NOTE]
+====
+The {product-title} {product-version} release introduces the `vcenters` and `failureDomains` fields for the `install-config.yaml` file. To define a zonal-enablement configuration for an environment with multiple datacentres running in a single VMware vCenter, you must set values for these fields. 
+====  
+
+The following example shows a VMware vSphere cluster configuration file with multiple data centers defined for a VMware vCenter. 
 
 [source,yaml]
 ----
 apiVersion: v1
-baseDomain: example.com <1>
-compute: <2>
-- hyperthreading: Enabled <3>
+baseDomain: example.com
+compute:
+- hyperthreading: Enabled
   name: worker
   replicas: 3
   platform:
-    vsphere: <4>
+    vsphere:
+      zones:
+        - <failure_domain1_name>
+        - <failure_domain2_name>
       cpus: 2
       coresPerSocket: 2
       memoryMB: 8192
       osDisk:
         diskSizeGB: 120
-controlPlane: <2>
-  hyperthreading: Enabled <3>
+controlPlane:
+  hyperthreading: Enabled
   name: master
   replicas: 3
   platform:
-    vsphere: <4>
+    vsphere:
+      zones:
+        - <failure_domain1_name>
+        - <failure_domain2_name>
       cpus: 4
       coresPerSocket: 2
       memoryMB: 16384
       osDisk:
         diskSizeGB: 120
 metadata:
-  name: cluster <5>
+  name: cluster
 ifdef::network[]
 networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <9>
+  networkType: OVNKubernetes
   serviceNetwork:
   - 172.30.0.0/16
 endif::network[]
 platform:
   vsphere:
-    vcenter: your.vcenter.server
-    username: username
-    password: password
-    datacenter: datacenter
-    defaultDatastore: datastore
-    folder: folder
-    resourcePool: resource_pool <6>
-    diskType: thin <7>
```

Comment:
Removed as it's no longer in the new default specification.

Reply:
is it still some where as optional?

Reply:
We do list all available config parameters in a [table](https://55767--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#installation-configuration-parameters_installing-vsphere-installer-provisioned-network-customizations).